salt/tests/requirements.in

```diff
@@ -0,0 +1,5 @@
+pytest ~= 4.3.0
+salt == 3000.3
+salttesting == 2018.9.21
```

Comment:
Maybe in future, we could consider adding flavour by using salts own pylint (saltpylint)

Reply:
Not really it's not a requirement for tests but for linting and it's part of this issue more #2649

Reply:
Ok agreed! so `saltpylint` should also be considered as a potential linting tool